.github/workflows/wheels.yml

```diff
@@ -85,6 +101,15 @@ jobs:
       - name: Setup Python
         uses: actions/setup-python@v2
 
+      # Comment out when numpy/scipy dependencies are on pypi
+      - name: Set pypi index for python dev
+        if: matrix.python == 310
+        run: |
+          echo "EXTRA_CIBW_ENVIRONMENT=PIP_EXTRA_INDEX_URL=https://pypi.anaconda.org/scipy-wheels-nightly/simple PIP_PRE=true" >> $GITHUB_ENV
+
+          # Do not use the dev version of pandas
+          echo "PANDAS_VERSION_SPECIFIER=<1.4" >> $GITHUB_ENV
```

Comment:
When a new version of Python come up, I expect the builds for the Python RC to be in https://pypi.anaconda.org/scipy-wheels-nightly/simple first, so setting `PIP_PRE=true` is required to automatically get the dev builds.

Currently there is a mix of Python 3.10 wheels on `scipy-wheels-nightly` and on pypi. numpy and pandas is on pypi and scipy is on `scipy-wheels-nightly`.